### ISSUE TITLE
Suggestions/improvements for 'Segments to export'

### ISSUE THREAD

**Issue by Wiethoofd** [mifi/lossless-cut#392]
The list of actions and shortcuts/interactions on a segment in the _Segments to export_ is currently very limited.

Currently you can only 'swap' segments you've created in the list of _Segments to export_, which gets quite frustrating if you created a new segment early in the file after creating a bunch that will come before and after it.

I have had faster and more successful swaps with naming my segments and using notepad++ to reorder the lines in the .csv and reloading the project file or the video in LLC than using the current 'reorder' swapping of segments.

There are barely any useful keyboard or mouse interactions with the created segments, a left-click selects the segment, which can then be deleted, swapped in order or renamed with the buttons at the bottom.

A possible nice addition would be a right-click menu and a (configurable) double-click action on a single segment.

* Single or double click could jump the timeline to the in-point of the segment
* Double click could play-back just the segment, exact in and out points and pause after playback
--
* Dragging and dropping to reorder (see [sortable.js Demo](http://sortablejs.github.io/Sortable/))
  * Possible modifier key support for multi-select, swap or default to simple reordering
--
* Right-click menu with various options:
  * Export single segment (#358) (with various output options)
  * Move segment up/down
  * Jump to in-/out-point
  * Label segment (same as currently when selecting and clicking the label button at the bottom)
  * Delete segment (with confirmation step that can be disabled in the settings)
--
* Preview all segments button at the bottom (#253)

**Comment by dquinton** [comment]
Good ideas!
I like the jump to start point in timeline - this could also be click on segment (panel or timeline) and a hotkeys such as < for start, and > for end.

Reordering would be good with up/down buttons at panel side (not by multiple right click menus)

Preview segments would be good, with an indication on timeline of position in playback.

**Comment by RobCrossRC** [comment]
I agree these are good suggestions, the jump to start of timeline is one that would be of great use to me, Ctrl + Home is a standard Windows command to do this in many programs.

**Comment by mifi** [comment]
next version will have a lot of this implemented!
